client/src/components/_common/Button/Button.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+
+import styles from './Button.module.css';
+
+export const TYPES = [
```

Comment:
@tacc-wbomar Do we need to worry about establishing UI for the other types in this array that I pulled [from the previous PR of this task](https://github.com/TACC/Core-Portal/pull/100/files) for this PR?

Reply:
**Nope.** In fact, you may _delete_ the unused ones.

```
  'success',
  'info',
  'warning',
  'danger',
  'link',
```

<details>
<summary>Details</summary>

I remember at least Matt S (lead designer) telling me we should not have a "rainbow" of button colors. I had pulled these types from messages in that previous PR.

</details>